MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -53,6 +56,8 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                         if (File.Exists(fontFile))
                             break;
                     }
+                    if (File.Exists(fontFile))
```

Comment:
that missing if is not related to the failure.. I should probably do that in a separate PR.